controllers/comments.controllers.js

```diff
@@ -41,3 +38,15 @@ exports.postCommentsByArticleId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentsByCommentId = (req, res, next) => {
+  const { comment_id } = req.params;
+  deleteComment(comment_id)
+    .then((result) => {
+      if (result.rowCount === 0) {
+        return res.status(404).json({ msg: 'comment does not exist' });
```

Comment:
You should handle this custom error in the model instead of the controller, rejecting the promise if the rows are empty. You may find that you need to put `RETURNING *` at the end of your SQL query in order to do this check in the model and reject the promise